### PR TITLE
Added no-var and camelcase eslint rules

### LINKED ISSUE #293: Enforce camelcase and var rules in eslint

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #293 

#### What's this PR do?

Adds eslint rules enforcing camelCase and the use of `let` or `const` over `var`. I also fixed the violations that it found.

#### Where should the reviewer start?

I guess just reviewing the changes overall.

#### How should this be manually tested?

Make sure the changes don't break anything!

### CODE REVIEW

File: static/js/util/api.js [hunk 1]
```diff
@@ -9,7 +9,7 @@ function getCookie(name) {
   if (document.cookie && document.cookie !== '') {
     let cookies = document.cookie.split(';');
 
-    for (var i = 0; i < cookies.length; i++) {
+    for (let i = 0; i < cookies.length; i++) {
```
Comment (noisecapella) [inline static/js/util/api.js]: Can we change this to `for (let cookie of cookies) {`? This code is adapted from the Django docs which are unaware of ES6

Comment (aliceriot) [inline static/js/util/api.js]: yes, I was pretty much blindly doing s/var/let/g haha

**Comment by noisecapella** [comment]
Looks good to me. Can we hold off on merging until after #241 is merged? We will need to fix some more lint errors at that point

**Comment by aliceriot** [comment]
sure, I'll update this PR with any fixes necessary after #241 has been merged

**Comment by aliceriot** [comment]
@noisecapella just updated this to fix the errors introduced by merging the education tab.

**Comment by noisecapella** [comment]
Looks good :+1:

**Pull request merged**